### PR TITLE
Made changes to Linux wiki. Removed some entries, expanded on others

### PR DESCRIPTION
**Pull request by coymeetsworld**
Made a fresh PR request only including updates to Linux page. Also made corrections/suggestions from  earlier PR request I made (which was now closed by me).

### CODE REVIEW

**Comment by Rafase282** [comment]
It looks good, I haven't check the actual content but I would prefer if you only keep the table of content and a brief history or definition in this page. The rest you can put in separated files that you would then link to this one.

This would make it more modular, make it easier for others to contribute on the same area via different topics and much more.

**Comment by coymeetsworld** [comment]
Closing this PR to break up the Linux wiki into smaller pages.

**Comment by coymeetsworld** [comment]
Put the 4 topics I had into their own wiki pages with the main Linux page having links to them through a Table of Contents. Let me know if the names of the files are good and also the content.

File: Linux-How-To-Install.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# How to pick a Linux distribution and install
```
Comment (Rafase282) [inline Linux-How-To-Install.md]: I would have named this one `Linux-Install.md`

**Comment by Rafase282** [comment]
Awesome job :+1:

File: Linux-How-To-Install.md (hunk 1 above)
Comment (coymeetsworld) [inline Linux-How-To-Install.md]: I don't have a problem with that, should I make that change while request is still open?

Comment (Rafase282) [inline Linux-How-To-Install.md]: Feel free to.

**Comment by Rafase282** [comment]
Squashed and merged without mentioning the extra commits that got leaked here: https://github.com/FreeCodeCamp/wiki/commit/04f7703ef5ea8640730778db8974eca280d2e330

**Pull request merged**